OsmAnd-api/src/net/osmand/aidlapi/IOsmAndAidlInterface.aidl

```diff
@@ -836,6 +837,13 @@ interface IOsmAndAidlInterface {
      */
     boolean removeAllActiveMapMarkers(in RemoveMapMarkersParams params);
 
+    /**
+     * Method to get position of various objects
+     *
+     * @params positionType (int) - type of position to get
+     */
+    boolean getPosition(in int positionType, out APosition position);
```

Comment:
New aidl methods must be added to the end of the file or following methods will be broken.

Reply:
Thanks, acknowledged )